pkg/cmd/server/start/start_master.go

```diff
@@ -378,7 +380,7 @@ func (m *Master) Start() error {
 	}
 
 	if m.api {
-		glog.Infof("Starting master on %s (%s)", m.config.ServingInfo.BindAddress, version.Get().String())
+		glog.Infof(MasterStartupMessage, m.config.ServingInfo.BindAddress, version.Get().String())
```

Comment:
I think needing to do this is an indication that scraping log output for regex matches on human readable messages isn't a very solid way to do diagnostics... I don't think externalizing the format strings for that purpose is a great idea. I would rather share a constant message ID that we use in the master log message and the diagnostics recognizes